lib/tasks/export.rake

```diff
@@ -0,0 +1,26 @@
+require 'csv'
+
+namespace :export do
+  def fetch_list(list_id)
+    begin
```

Comment:
You actually don't need this extra `begin` and `end`, the `rescue` can go right in the same level as the method:

```ruby
def method
  ...
rescue ...
  ...
end
```

Reply:
The flexibility of Ruby syntax continues to surprise me